ting_infomedia.module

```diff
@@ -27,8 +27,19 @@ function ting_infomedia_menu() {
  * Return a render array
  */
 function ting_infomedia_view($id) {
+  global $user;
+  if( !ding_user_is_provider_user($user)){
```

Comment:
@kasperg : it's a requirement that the user is a user in the library system (the credentials will be checked in the Infomedia webservice). So, unless editorial users are also library users (and use the same credentials for login), they won't be permitted to view the article.